remote_syslog.go

```diff
@@ -53,6 +54,11 @@ func tailOne(file string, excludePatterns []*regexp.Regexp, logger *syslog.Logge
 // Tails files speficied in the globs and re-evaluates the globs
 // at the specified interval
 func tailFiles(globs []string, excludedFiles []*regexp.Regexp, excludePatterns []*regexp.Regexp, interval RefreshInterval, logger *syslog.Logger, severity syslog.Priority, facility syslog.Priority, poll bool) {
+	if len(globs) == 0 {
```

Comment:
Agreed that we want to error and exit if no files have been specified. My suggestion was to run the check earlier so it can be printed when daemonized. 

I agree with @wagerlabs that it's likely worth merging everything early on.